### DISCUSSION TITLE
Disable /tps timer

### DISCUSSION THREAD

**Discussion by BlueSoapTurtle** [Zrips/CMI#2287] [Feature Requests]
**Description of feature/improvement request:**
It seems like there's a task running every tick to calculate the tps
I'd prefer using the default /tps command in spigot/paper, so is it possible to not run the task?
CMI::Task: com.Zrips.CMI.utils.Lag (interval:1)count(2046)  total(0.05% 0.119s, 0.08% of tick)avg(0.06ms per - 0.04ms/0.70 per tick)

since it's using 0.04ms to run every tick, and I'd rather just save that part
---
### Detailed explanation of what it should do, when and in what situation: ###
add config option like this:
enable-tps-logging: false

### Basic explanation as to why this feature is needed and/or useful to have: ###
Setting it to false will make the tps task not even be started, the com.Zrips.CMI.utils.Lag

**Comment by Zrips** [comment]
its not really a option we can have for it, as its been used in multiple areas and not only tps command. What i can do is to lower tps check, instead of recording each tick, we can record each second, which should be enough for our needs without compromising its behavior while lowering load 20 times.
